astro/src/content/docs/release-notes/index.mdx

```diff
@@ -339,6 +339,8 @@ Update dependencies.
 * Applications may now be designated as third-party applications. In addition to the prompting for authorization that comes with the `Custom OAuth Scopes` feature (see above), limitations are being added to how third-party applications may interact with FusionAuth.
   * Resolves [GitHub Issue #2723](https://github.com/FusionAuth/fusionauth-issues/issues/2723)
 * Applications can now be configured to prompt users to grant consent to requested OAuth scopes using the `/oauth2/consent` [themed](/docs/customize/look-and-feel/) page. See the OAuth [Scopes](/docs/lifecycle/authenticate-users/oauth/scopes) for more detail.
+
+  The new consent prompt themed page requires the `scopeConsentField` macro and `resolveScopeMessaging` function to be defined in the _Helpers_ template in order to render scope consent form fields. These _must_ be added to a custom theme in order for it to function. See [theme upgrade notes](/docs/customize/look-and-feel/upgrade#version-1500) for details on updating an existing theme with the new macro and function.
```

Comment:
What do you think about moving this paragraph up to the top of the `1.50.0` section and putting it in a `<ThemeUpdateRequired>` to make it more visible? (`<ThemeUpdateRequired>` doesn't exist yet)